src/openforms/js/components/form/edit/tabs.js

```diff
@@ -87,40 +88,15 @@ const CHOICES_BASIC = {
   key: 'basic',
   label: 'Basic',
   components: [
-    ...BASIC.components,
-    {
-      type: 'datagrid',
-      input: true,
-      label: 'Values',
-      key: 'values',
-      tooltip:
-        'The values that can be picked for this field. Values are text submitted with the form data. Labels are text that appears next to the radio buttons on the form.',
-      weight: 10,
-      reorder: true,
-      defaultValue: [{label: '', value: ''}],
-      components: [
-        {
-          label: 'Label',
-          key: 'label',
-          input: true,
-          type: 'textfield',
-          // Needed to distinguish from the label of the component, since both have key 'label'.
-          // Issue #1422
-          isOptionLabel: true,
-        },
-        {
-          label: 'Value',
-          key: 'value',
-          input: true,
-          type: 'textfield',
-          allowCalculateOverride: true,
-          calculateValue: {_camelCase: [{var: 'row.label'}]},
-          validate: {
-            required: true,
-          },
-        },
-      ],
-    },
+    LABEL_REQUIRED,
+    KEY,
+    DESCRIPTION,
+    PRESENTATION,
+    MULTIPLE,
+    HIDDEN,
+    CLEAR_ON_HIDE,
+    IS_SENSITIVE_DATA,
```

Comment:
Would people miss the combination "manual" and setting the default value?
I agree leaving it in, as is, is way too confusing.

Reply:
It is still possible to add the default value, but only in the manual case. So I moved the field inside the 'options' panel and added a condition to it!

![Screenshot from 2023-01-26 10-59-58](https://user-images.githubusercontent.com/19154114/214808051-4b74e8b0-d10c-4efe-b6ad-665517bfc11f.png)

Reply:
Oh, great. I didn't see that.